auslib/test/admin/views/test_rules.py

```diff
@@ -505,11 +536,14 @@ def testDeleteRuleOutdatedData(self):
         self.assertEquals(ret.status_code, 400, msg=ret.data)
 
     def testDeleteRuleByAlias(self):
-        ret = self._delete('/rules/frodo', qs=dict(data_version=1))
+        ret = self._delete('/rules/frodo', qs={"data_version": 1})
         self.assertEquals(ret.status_code, 200, msg=ret.data)
 
     def testDeleteRule404(self):
         ret = self._delete("/rules/112")
+        # Missing data_version field
+        self.assertEquals(ret.status_code, 400)
```

Comment:
Same thing goes here -- this test's purpose isn't to verify the data_version handling, it's to verify 404 handling, so get rid of the extra delete() and pass data_version in the existing one.